docs/source/mistral.rst

```diff
@@ -175,6 +175,20 @@ Since there are multiple workflows defined in this workbook, the workflow author
 
 To test out this workflow, save the metadata file to /opt/stackstorm/packs/examples/actions/ and the workflow file to /opt/stackstorm/packs/examples/actions/workflows. Run ``st2 action create /opt/stackstorm/packs/examples/actions/mistral-workbook-complex.yaml`` to create the action and run ``st2 run examples.mistral-workbook-complex vm_name="vmtest1" -a`` to test.
 
+Validation
++++++++++++++++++++
+The Mistral CLI ships with the capability of performing high-level sanity checks
+of Mistral workflow YAML files.
+
+Example:
+.. code-block:: bash
+   mistral workflow-validate /path/to/workflow.yaml
+
```

Comment:
You could add something like:

For testing mistral workbooks (nested mistral workflows) use the following command:
`mistral worbook-validate /path/to/workflow.yaml'